### DISCUSSION TITLE
"Helpful" event not registering in GA4

### DISCUSSION THREAD

**Discussion by TheCaptNink** [google/docsy#1426] [Q&A]
Hi all,

I've switched our analytics account from UA to GA4, as per Google's insistence. When we were using UA, the "Helpful" event that is registered when a user clicks one of the Feedback buttons was working as described in the Docsy documentation. Now that I've changed to GA4, that's no longer happening. 

I replaced our UA ID in the config.toml with the new GA4 tag and pageviews, clicks, etc are all registering, but not "Helpful".

I'm a Tech Writer and not a coding or analytics genius by any stretch, but I feel like I'm missing something with this GA4 garbage. Any help would be appreciated.

**Comment by chalin** [comment]
Hi. This is a known problem:

- #1302

Until a fix lands here, you can try the workaround described at, https://github.com/google/docsy/issues/1302#issuecomment-1301550895. I hope that this helps.

**Reply by TheCaptNink** [reply]
That does help! Thank you for the reply! Much appreciated!

**Comment by chalin** [comment]
FYI, the following will be supported as part of the next release (0.8.0):

- #1302
